src/components/Pages/ManageWiki/Features/Lexeme.vue

```diff
@@ -53,41 +54,47 @@ export default {
   name: 'Lexeme',
   props: ['wikiId', 'setting'],
   data () {
-    return {
+    const data = {
       valid: false,
       dialog: false,
       userConfirmationString: 'wbstack-important-change',
-      rules: {
-        match: (value) =>
-          value === this.userConfirmationString || 'Please enter the confirmation'
```

Comment:
Second problem was that `this` here would be the Vue component, not the `data` object, thus returning `undefined` instead of the value of `userConfirmationString`

Also the comparison is borked and would return either `true|'Please enter the confirmation'` instead of `true|false`

Reply:
I'm very unclear how this ever really worked